tests/rtl/hardware_test.py

```diff
@@ -385,6 +388,73 @@ def test_hardware_axpy_double_pump_vec4():
     return test_hardware_axpy_double_pump(veclen=4)
 
 
+@rtl_test()
+def test_hardware_vadd_temporal_vectorization():
+    # TODO !!!!! THIS TEST STALLS WITH VITIS 2021.2 !!!!!
+    # But it works fine for 2020.2
+    with dace.config.set_temporary('compiler', 'xilinx', 'frequency', value='"0:300\\|1:600"'):
+        # Generate the test data and expected results
+        size_n = 1024
+        veclen = 4
+        N = dace.symbol('N')
+        N.set(size_n)
+        x = np.random.rand(N.get()).astype(np.float32)
+        y = np.random.rand(N.get()).astype(np.float32)
+        result = np.zeros(N.get(), dtype=np.float32)
+        expected = x + y
+
+        # Generate the initial SDFG
+        def np_vadd(x: dace.float32[N], y: dace.float32[N]):
+            return x + y
+
+        sdfg = dace.program(np_vadd).to_sdfg()
+
+        # Remove underscores as Xilinx does not like them
```

Comment:
Why is this needed here in a test?
Should the user do the same in her own program?

Reply:
This has been fixed, so it's now handled in the Xilinx codegen. So no, it should not have to be done by the user.